src/server/app/admin/models/user.server.model.js

```diff
@@ -153,6 +153,11 @@ let UserSchema = new GetterSchema({
 		default: Date.now,
 		get: util.dateParse
 	},
+	messagesViewed: {
```

Comment:
should we also add this field to the audit copy? a user update gets audited when the messagesViewed timestamp is updated, but in the audit entry it looks like there are no changes